x/claims/keeper/msg_server.go

```diff
@@ -35,6 +42,26 @@ func (k *Keeper) UpdateParams(goCtx context.Context, req *types.MsgUpdateParams)
 	}
 
 	ctx := sdk.UnwrapSDKContext(goCtx)
+
+	// Validate the requested authorized channels
+	authorizedChannels := req.Params.AuthorizedChannels
+	for _, channelName := range authorizedChannels {
+		if err := host.ChannelIdentifierValidator(channelName); err != nil {
+			return nil, errorsmod.Wrapf(err,
+				"invalid authorized channel contained in the request to update the claims parameters: %s",
+				channelName,
+			)
+		}
+
+		channelID := strings.TrimPrefix(channelName, channeltypes.ChannelPrefix)
+		if _, found := k.channelKeeper.GetChannel(ctx, transfertypes.PortID, channelID); !found {
```

Comment:
add a comment saying that we're ussing the default PortID and this will fail if the port is different